test/source/tests/tests/compose.ts

```diff
@@ -550,14 +550,24 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       expect(await body.$eval('#input_text img', el => el.getAttribute('src'))).to.eq(imgBase64);
     }));
 
-    ava.default('compose - saving and rendering a draft with RTL text', testWithBrowser('compatibility', async (t, browser) => {
+    ava.default('compose - saving and rendering a draft with RTL text (plain text)', testWithBrowser('compatibility', async (t, browser) => {
       let composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
-      const subject = `saving and rendering a draft with RTL text`;
+      const subject = `saving and rendering a draft with RTL text (plain text)`;
+      await ComposePageRecipe.fillMsg(composePage, { to: 'human@flowcrypt.com' }, subject);
```

Comment:
Should make sure to explicitly set `richtext: false`. That way when we later switch default behavior, this will continue testing plain text and not rich text.

Reply:
Done :+1: